lib/cylc/scheduler.py

```diff
@@ -117,6 +117,7 @@ class Scheduler(object):
     EVENT_STARTUP = 'startup'
     EVENT_SHUTDOWN = 'shutdown'
     EVENT_TIMEOUT = 'timeout'
+    EVENT_INACTIVITY_TIMEOUT = 'inactivity'
```

Comment:
I think I'd prefer the event to be 'inactive' rather than 'inactiviity'.  Opinions?